packages/vkui/src/components/ModalPage/ModalPage.tsx

```diff
@@ -129,6 +136,7 @@ export const ModalPage = ({
           className={styles['ModalPage__in-wrap']}
           style={{
             maxWidth: typeof size === 'number' ? size : undefined,
+            ...(height ? { height } : null),
```

Comment:
Вроде height как undefined должен же принимать?

```suggestion
            height,
```

Reply:
> Вроде height как undefined должен же принимать?

Точно, ты прав!
⚡ https://github.com/VKCOM/VKUI/pull/5830/commits/7a6bca02f13eaecec5291ccf5727687ffaf73c1c